index.js

```diff
@@ -102,6 +103,7 @@ function mobileMenuOption(event, id) {
   const linkTo = document.getElementById(id);
   linkTo.scrollIntoView();
 }
+// Shows project details popup
```

Comment:
- [OPTIONAL ] I think the color you have chosen is meaningful but adjust it a bit, as you can see, this red color is too spicy, Please take a look at it.
- 
<img width="445" alt="PIVK" src="https://user-images.githubusercontent.com/79058364/156457483-7be93ad2-16ae-4034-9d19-c4f76b934e13.png">